src/control/system/member.go

```diff
@@ -180,14 +182,19 @@ func (sm *Member) UnmarshalJSON(data []byte) error {
 }
 
 func (sm *Member) String() string {
-	return fmt.Sprintf("%s/%d", sm.Addr, sm.Rank)
+	return fmt.Sprintf("%s/%d/%s", sm.Addr, sm.Rank, sm.State())
 }
 
 // State retrieves member state.
 func (sm *Member) State() MemberState {
 	return sm.state
 }
 
+// Equals verifies equivalency.
+func (sm *Member) Equals(m *Member) bool {
+	return reflect.DeepEqual(*sm, *m)
+}
+
```

Comment:
Seems like this might be better as a test helper... Generally wary of using the reflect package in production.